types/react-calendar-heatmap/index.d.ts

```diff
@@ -1,30 +1,42 @@
-// Type definitions for react-calendar-heatmap 1.6
+// Type definitions for react-calendar-heatmap 1.8
 // Project: https://github.com/patientslikeme/react-calendar-heatmap
 // Definitions by: Keisuke Kan <https://github.com/9renpoto>
+//                 Seungbin Oh <https://github.com/sboh1214>
```

Comment:
Are you intending to become an owner of this types package, or were you looking for where to add credit for your changes? If the latter, you shouldn't add your name here. https://github.com/DefinitelyTyped/DefinitelyTyped#definition-owners